src/js/plugins/plugin.chapters.js

```diff
@@ -181,3 +227,42 @@ BookReader.prototype.getOpenLibraryRecord = function () {
       }
     });
 }
+
+// Extend buildMobileDrawerElement with table of contents list
+BookReader.prototype.buildMobileDrawerElement = (function (super_) {
+  return function () {
+    const $el = super_.call(this);
+    if (this.options.enableChaptersPlugin) {
+      $el.find('.BRmobileMenu__moreInfoRow').after($(`
+        <li class="BRmobileMenu__tableContents" data-event-click-tracking=”BRSidebar|TOCPanel>
+            <span>
+                <span class="DrawerIconWrapper"></span>
+                Table of Contents
+            </span>
+            <div>
```

Comment:
This should let us avoid using `mm-4` as a selector:

```suggestion
            <div class="BRmobileMenu__tableContentsPanel">
```

Reply:
done